nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -306,29 +306,29 @@ def export_model(self, model_path, calibration_path=None, onnx_path=None, input_
             if hasattr(module, 'weight_scale') or hasattr(module, 'input_scale') or hasattr(module, 'output_scale'):
                 calibration_config[name] = {}
             if hasattr(module, 'weight_scale'):
-                calibration_config[name]['weight_bit'] = 8
+                calibration_config[name]['weight_bits'] = 8
                 val = float(module.weight_scale * module.weight_qmax)
                 calibration_config[name]['tracked_max_weight'] = val
                 calibration_config[name]['tracked_min_weight'] = -val
                 calibration_config[name]['tracked_weight_qmin'] = -127
                 calibration_config[name]['tracked_weight_qmax'] = 127
```

Comment:
it's better to use `tracked_qmin_weight`, to be consistent with `tracked_qmin_output`

Reply:
Ok, done.